index.html

```diff
@@ -411,9 +411,15 @@
     }
 
     scheduleElement.entries = todaySchedule;
-
+    if (todaySchedule.length == 0) { // If there are no classes
+      var imageNum = Math.floor((Math.random() * 6) + 1);
```

Comment:
"6" should really be some sort of NUM_NOSCHOOL_IMAGES constant, rather than a confusing magic number